src/server/ApiRoutes/Hold.js

```diff
@@ -251,83 +251,6 @@ function confirmRequestServer(req, res, next) {
     });
 }
 
-/**
- * newHoldRequestServer(req, res, next)
- * The function to return the bib and item data with its delivery locations to hold request page.
- *
- * @param {req}
- * @param {res}
- * @param {next}
- * @return {function}
- */
-function newHoldRequestServer(req, res, next) {
-  const bibId = req.params.bibId || '';
-  const loggedIn = User.requireUser(req, res);
-  const error = req.query.error ? JSON.parse(req.query.error) : {};
-  const form = req.query.form ? JSON.parse(req.query.form) : {};
-
-  if (!loggedIn) return false;
-
-  const patronId = req.patronTokenResponse.decodedPatron.sub || '';
-  let barcode;
-
-  // Retrieve item
-  return Bib.fetchBib(
-    bibId,
-    (bibResponseData) => {
-      barcode = LibraryItem.getItem(bibResponseData, req.params.itemId).barcode;
-
-      getDeliveryLocations(
```

Comment:
This confused me, how is this then being populated on the hold page?

Reply:
This is all being handled by `newHoldRequestAjax` now. For client side calls, `dataLoaderUtil` will trigger a call to the `/api/hold/request` route, which resolves to `newHoldRequestAjax`. For server side calls, `dataLoaderUtil` calls `newHoldRequestAjax` directly.